src/components/Garden/Footer.js

```diff
@@ -60,17 +60,18 @@ const defaultFooterData = {
 function Footer() {
   const theme = useTheme()
   const { below } = useViewport()
-  const compactMode = below('large')
+  const compactMode = below('medium')
   const [footerData, setFooterData] = useState(defaultFooterData)
 
   const { connectedGarden } = useGardens()
 
   useEffect(() => {
     if (connectedGarden) {
-      const { links, logo, token, wrappableToken } = connectedGarden
+      // eslint-disable-next-line camelcase
+      const { links, logo_type, token, wrappableToken } = connectedGarden
       setFooterData({
         links,
-        logo,
+        logo: logo_type,
```

Comment:
Where has been decided to use the logo type? Doesn´t it look too big for a footer?

Reply:
Fiore told me she design using the logo_type in the footer. It will only show on tablet and desktop sizes. So not worried to much about the size. We can consider to reduce the scale.